fastapi-google-stt/app/db/reset_database.py

```
@@ -0,0 +1,16 @@
+from sqlalchemy.ext.asyncio import create_async_engine
+
+from app.db.database import Base
+
+import app.db.entity
```

Ruff (0.8.2)

`app.db.entity` imported but unused

Remove unused import: `app.db.entity`

(F401)